backend/routes/content.py

```
@@ -301,17 +307,17 @@ def mark_module_complete_and_open_next(module_id, user_id):
         user_module = UserModule.query.filter_by(
             module_id=module_id, user_id=user_id
         ).first()
-        # Special case: add the first module to the UserModule table
-        if order == 1 and user_module is None:
+
+        # TODO: refactoring -- if I create the user_module here, I do not need to keep checking for it later
+        if user_module is None:
+            logger.debug(f"Adding first module {module_id} to UserModule table")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)